### DISCUSSION TITLE
Settings: always on top

### DISCUSSION THREAD

**Discussion by gesicki** [sircharlo/meeting-media-manager#371] [Ideas] [answered]
Settings

I miss the option in the program window settings: always on top

Regards

Mariusz Gęsicki
email: gesicki@wp.pl

**Comment by sircharlo** [comment, marked as answer]
What are some cases in which the media window should **not** be on top?
I'm just debating whether such a setting really needs to exist, as right now the media window is always on top of other windows automatically. When would this not be desirable?

**Comment by mtdvlpr** [comment]
Whenever you want to display something else on the screen (like Zoom) you can just turn off the media window.

**Reply by sircharlo** [reply]
Exactly, using <kbd>Alt</kbd> <kbd>Z</kbd>...
